client/components/rewind-credentials-form/index.jsx

```diff
@@ -152,6 +152,11 @@ export class RewindCredentialsForm extends Component {
 		return (
 			<div className="rewind-credentials-form">
 				<QueryRewindState siteId={ siteId } />
+				<div className="rewind-credentials-form__instructions">
+					{ translate(
+						'Please get your credentials from your hosting provider. Their website should explain how to get or create the credentials you need.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'You can normally get your credentials from your hosting provider. Their website should explain how to get or create the credentials you need.' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).